[ZT] Gateway Fixes (#3474)
cloudflare/cloudflare-docs: products/cloudflare-one/src/content/setup/index.md

1. On your Account Home in the Cloudflare Dashboard, click on the Zero Trust icon.

1. On the onboarding screen, choose a [team name](/glossary#team-name).

> 1. Complete your onboarding by selecting a subscription plan and entering your payment details. If you chose the **Zero Trust Free plan**, please note this step is still needed, but you won’t be charged.

Reviewer: ```suggestion
1. Complete your onboarding by selecting a subscription plan and entering your payment details. If you chose the **Zero Trust Free plan**, please note this step is still needed, but you will not be charged.
```
